Formula/freetype.rb

```diff
@@ -34,6 +34,10 @@ def install
     system "./configure", "--prefix=#{prefix}", "--without-harfbuzz"
     system "make"
     system "make", "install"
+
+    %W[#{bin}/freetype-config #{lib}/pkgconfig/freetype2.pc].each do |f|
+      inreplace f, prefix, opt_prefix
+    end
```

Comment:
I don't have an objection to using the block format to fix multiple files personally, but if you want to avoid that & avoid blowing past the 80 character mark your initial preference for just having two `inreplace` lines is 👍.